nilearn/_utils/tests/test_param_validation.py

```diff
@@ -66,7 +66,7 @@ def test_check_threshold():
 
 def test_get_mask_volume():
     # Test that hard-coded standard mask volume can be corrected computed
-    if os.path.isfile(mni152_brain_mask):
+    if Path.is_file(mni152_brain_mask):
```

Comment:
for `is_file()` you must also use

```suggestion
    if Path(mni152_brain_mask).is_file():
```

Reply:
Same with this one. Is there some way to fin through the codes where Path.is_file() is used and change it